src/StaticFAQ/ArticleFeedback/ScreenInput.js

```diff
@@ -96,19 +115,24 @@ class ScreenInput extends React.Component<Props, State> {
           <div className="question">
             <Text>What problem were you trying to solve?</Text>
           </div>
-          <div className="inputArea">
+          <div className={`inputArea ${error ? 'invalid' : ''}`}>
             <textarea
               data-gramm_editor="false"
               onChange={this.handleChange}
               value={this.state.comment}
-              required
             />
+            {error ? (
+              <Text size="small">You haven't written any feedback.</Text>
+            ) : null}
           </div>
           <div className="button">
             <Button title="Submit" onClick={() => {}} width={100} />
           </div>
         </form>
         <style jsx>{style}</style>
+        <style jsx global>
```

Comment:
I know it's annoying not to be able to change the color of `Text` to red cause of Orbit props values :(
However I think it's better to just use a plain `<p>` tag instead of `global` styles. @jalbertsr would you agree?

Reply:
yes, true, agree